File: swiftSchema/ClientNotification.swift
import Foundation

struct ClientNotification: Codable, Identifiable, Equatable {
	var id = UUID()

	var notification_id: Int?
	var image_code: Int?
	var header: String?
	var body: String?
	var time_stamp: Int?
	var action_payload: String?
	var action: Int?
	var viewed: Int?
	var free_text: String?

	private enum CodingKeys: String, CodingKey {
		case notification_id, image_code, header, body, time_stamp, action_payload, action, viewed, free_text
	}
}

File: klaxonModels/ClientNotification.kt
package io.homemetrics.homemetrics.schemas

import com.beust.klaxon.Klaxon

data class ClientNotification(
	val notification_id: Integer?,
	val image_code: Integer?,
	val header: String?,
	val body: String?,
	val time_stamp: Integer?,
	val action_payload: String?,
	val action: Integer?,
	val viewed: Integer?,
	val free_text: String?
)